.github/workflows/build-streams.yaml

```diff
@@ -1,6 +1,7 @@
 name: build sentry_streams
 
 on:
+  pull_request:
```

Comment:
Is this intentional or is it a left over ?

Reply:
this is intentional to test the release build within this PR and show that it works.